Orange/widgets/cpewidget.py

```diff
@@ -32,7 +36,11 @@ def sync_properties(self):
         # to the widget
```

Comment:
Ok, we're getting there...
I think the line above this also needs a translation from NoneUIValues to None, though.